### ISSUE TITLE
Dropdown accessibility issues

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1603]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

Menu items are called "Basic uncontrolled example option" and not the name of the item
Expect: The name of the item should be the visible label

**Comment by scriby** [comment]
This may be related to https://github.com/OfficeDev/office-ui-fabric-react/issues/1449#issuecomment-297196283

**Comment by betrue-final-final** [comment]
Thanks @scriby, that could be it.

**Comment by betrue-final-final** [comment]
No repro. There's an issue where using the down arrow doesn't announce a state change, but a different bug.
